Makefile

```diff
@@ -88,7 +88,7 @@ endif
 glide.lock: glide.yaml
 	echo "Glide.yaml has changed, updating glide.lock"
 	if [ "$(GLIDE_HOME)" ]; then mkdir -p "$(GLIDE_HOME)"; fi
-	$(GLIDE) update -v
+	timeout -v 20m $(GLIDE) update -v || timeout -v 20m $(GLIDE) update -v
```

Comment:
John, changes are required in the install, Install is the one where CentOS ci is failing. I raised a test PR to see if that works. If it works then we can make the same changes here and get this merged.